diff -urN aa/skijump-0.2.0/Makefile.am skijump-0.2.0/Makefile.am
--- aa/skijump-0.2.0/Makefile.am	2004-07-23 10:20:02.000000000 +0200
+++ skijump-0.2.0/Makefile.am	2004-07-27 01:02:35.280072352 +0200
@@ -5,7 +5,7 @@
 CPPFLAGS += -DDATADIR=\"${skijumpdatadir}/\" -DCFGDIR=\"${skijumpcfgdir}/\" -DREPLAYDIR=\"${skijumpreplaydir}/\"
 
 install-data-local:
-	chgrp games ${execprefix}${bindir}/skijump
-	chmod g+s ${execprefix}${bindir}/skijump
-	chgrp -R games ${skijumpcfgdir}
-	chmod -R g+rw ${skijumpcfgdir}
+	chgrp games $(DESTDIR)${execprefix}${bindir}/skijump
+	chmod g+s $(DESTDIR)${execprefix}${bindir}/skijump
+	chgrp -R games $(DESTDIR)${skijumpcfgdir}
+	chmod -R g+rw $(DESTDIR)${skijumpcfgdir}
